eth/beacon/types/blocks.py

```diff
@@ -104,6 +102,11 @@ def hash(self) -> Hash32:
             self._hash = blake(rlp.encode(self))
         return self._hash
 
+    @property
+    def root(self) -> Hash32:
```

Comment:
- option 1: replace `def hash` with `def root`.
- option 2: have `def root` as the alias
- option 3: have `def hash` as the alias

Not quite understand the rationale of choosing option 2 if `root` is the preferred, could you elaborate?

Reply:
1. For convenience 😅 
2. BTW, there will be [tree_hash_root](https://github.com/ethereum/eth2.0-specs/blob/master/specs/simple-serialize.md#tree-hash) for SSZ object serialization.